pkg/storage/storagewrappers/caching.go

```diff
@@ -52,14 +52,14 @@ func (c *cachedOpenFGADatastore) ReadAuthorizationModel(ctx context.Context, sto
 }
 
 // FindLatestAuthorizationModelID returns the last model `id` written for a store.
-func (c *cachedOpenFGADatastore) FindLatestAuthorizationModelID(ctx context.Context, storeID string) (string, error) {
-	v, err, _ := c.lookupGroup.Do(fmt.Sprintf("FindLatestAuthorizationModelID:%s", storeID), func() (interface{}, error) {
-		return c.OpenFGADatastore.FindLatestAuthorizationModelID(ctx, storeID)
+func (c *cachedOpenFGADatastore) FindLatestAuthorizationModelID(ctx context.Context, storeID string) (*openfgav1.AuthorizationModel, error) {
```

Comment:
Ohh this is such a sneaky bug..

your change:
<img width="1102" alt="image" src="https://github.com/openfga/openfga/assets/5374887/45334890-5f01-4413-9528-34ae11c80bfb">

but we should update the name of the method too:

<img width="1119" alt="image" src="https://github.com/openfga/openfga/assets/5374887/68ccbe64-080e-46be-ac2a-3fb9b9b7b464">

Reply:
fixed